<ui version="4.0" >
 <class>katlasToolBox</class>
 <widget class="QWidget" name="katlasToolBox" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>591</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>5</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <property name="layoutDirection" >
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QToolBox" name="toolBox" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>150</width>
       <height>410</height>
      </size>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="toolBoxTab1" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>336</width>
        <height>529</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Navigation</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="5" column="0" >
          <widget class="Line" name="line_2_2" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="label_2_2" >
           <property name="text" >
            <string>&amp;Search</string>
           </property>
           <property name="buddy" >
            <cstring>searchLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLineEdit" name="searchLineEdit" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>5</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="SearchListView" name="locationListView" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>5</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="focusPolicy" >
            <enum>Qt::WheelFocus</enum>
           </property>
           <property name="horizontalScrollBarPolicy" >
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="editTriggers" >
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors" >
            <bool>false</bool>
           </property>
           <property name="iconSize" >
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="layoutMode" >
            <enum>QListView::Batched</enum>
           </property>
           <property name="viewMode" >
            <enum>QListView::ListMode</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>0</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>0</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="1" column="0" >
            <widget class="QToolButton" name="moveLeftButton" >
             <property name="text" >
              <string>...</string>
             </property>
             <property name="icon" >
              <iconset resource="marble.qrc" >:/icons/arrow-left.png</iconset>
             </property>
            </widget>
           </item>
           <item row="5" column="1" >
            <widget class="QToolButton" name="zoomOutButton" >
             <property name="text" >
              <string>...</string>
             </property>
             <property name="icon" >
              <iconset resource="marble.qrc" >:/icons/zoom-out.png</iconset>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QToolButton" name="moveUpButton" >
             <property name="text" >
              <string>...</string>
             </property>
             <property name="icon" >
              <iconset resource="marble.qrc" >:/icons/arrow-up.png</iconset>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QToolButton" name="goHomeButton" >
             <property name="text" >
              <string>...</string>
             </property>
             <property name="icon" >
              <iconset resource="marble.qrc" >:/icons/go-home.png</iconset>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <widget class="QToolButton" name="moveDownButton" >
             <property name="text" >
              <string>...</string>
             </property>
             <property name="icon" >
              <iconset resource="marble.qrc" >:/icons/arrow-down.png</iconset>
             </property>
            </widget>
           </item>
           <item row="3" column="1" >
            <widget class="QToolButton" name="zoomInButton" >
             <property name="text" >
              <string>...</string>
             </property>
             <property name="icon" >
              <iconset resource="marble.qrc" >:/icons/zoom-in.png</iconset>
             </property>
            </widget>
           </item>
           <item row="4" column="1" >
            <widget class="QSlider" name="zoomSlider" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>1</hsizetype>
               <vsizetype>7</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimum" >
              <number>1000</number>
             </property>
             <property name="maximum" >
              <number>2500</number>
             </property>
             <property name="value" >
              <number>1050</number>
             </property>
             <property name="sliderPosition" >
              <number>1050</number>
             </property>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="tickPosition" >
              <enum>QSlider::TicksAbove</enum>
             </property>
             <property name="tickInterval" >
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="6" column="1" >
            <widget class="QFrame" name="m_pSpacerFrame" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>5</hsizetype>
               <vsizetype>0</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>16</width>
               <height>0</height>
              </size>
             </property>
             <property name="frameShape" >
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Plain</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="2" >
            <widget class="QToolButton" name="moveRightButton" >
             <property name="text" >
              <string>...</string>
             </property>
             <property name="icon" >
              <iconset resource="marble.qrc" >:/icons/arrow-right.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>0</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="toolBoxTab2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>336</width>
        <height>529</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Map View</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="KAtlasThemeSelectView" name="katlasThemeSelectView" >
         <property name="autoFillBackground" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchListView</class>
   <extends>QListView</extends>
   <header>searchlistview.h</header>
  </customwidget>
  <customwidget>
   <class>KAtlasThemeSelectView</class>
   <extends>QListView</extends>
   <header>katlasthemeselectview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchLineEdit</tabstop>
  <tabstop>locationListView</tabstop>
  <tabstop>moveUpButton</tabstop>
  <tabstop>moveLeftButton</tabstop>
  <tabstop>goHomeButton</tabstop>
  <tabstop>moveRightButton</tabstop>
  <tabstop>moveDownButton</tabstop>
  <tabstop>zoomInButton</tabstop>
  <tabstop>zoomSlider</tabstop>
  <tabstop>zoomOutButton</tabstop>
 </tabstops>
 <resources>
  <include location="marble.qrc" />
 </resources>
 <connections/>
</ui>
